risedev.yml

```diff
@@ -92,6 +92,17 @@ profile:
       - use: compute-node
       - use: frontend
 
+  # You can use this in combination with the virtual compactor
+  # provided in https://github.com/risingwavelabs/risingwave-extensions
+  dev-compactor:
+    steps:
+      - use: minio
+      - use: meta-node
+      - use: compute-node
+      - use: frontend
+      - use: compactor
+        user-managed: true
```

Comment:
Little confuse about why this can used for dev compactor.

Reply:
`user-managed` just means that meta and CN expects there to be a compactor, but that this compactor will be started by the user. You could start your compactor in debug mode OR in our case you start a virtual compactor with the params defined in https://github.com/risingwavelabs/risingwave-extensions/pull/36.

Reply:
Get